_posts/2016/06/2016-06-31-brazil.md

```diff
@@ -0,0 +1,95 @@
+date: 2016-06-31
+time: "10:00:00"
+category: ["workshops"]
+---
+In Brazil, like in many other countries that we are starting to run workshops,
+there are many open questions that we need to answer such as
+
+- how do we advertize our workshops?
+- do we charge a registration fee? If yes, how much do we charge?
+- how do we create a local community after the workshop?
+- how do we get new partners after create a local community?
+
+Thanks to the work of many instructors like
+Anelda van der Walt,
+Belinda Weaver,
+Bill Mills,
+Lex Nederbragt,
+Selene L. Fernandez-Valverde, and
+Tiffany Timbers
+we are getting some hypothesis to the questions above across the world.
+In terms of Brazil, the three workshops that we ran in May added more data
+to help answering the questions.
+
+## Workshop in Florianópolis
+
+[This](https://rgaiacs.github.io/2016-05-16-scipyla/) was the [second](https://dbarneche.github.io/2014-12-11-ufsc/)
+workshop that we run in Florianópolis
+with more than one year between each.
+
+Our 20 seats sold out in a few days (we didn't charge anything) but only four students attended all the sessions making this the workshop in Brazil with the smallest number of attendees. The low number of attendees can be due our workshop ran at the same time of others activities of [SciPy Latin America 2016](http://scipyla.org/conf/2016/) but I believe that the main reason was the missing of a local champion that motivate people to attend the workshop (in 2014 we had Diego Barneche Rosado).
+
+## Workshop in Campinas
+
+[This](https://rgaiacs.github.io/2016-05-23-unicamp/) also was the [second](https://r-gaia-cs.github.io/2015-06-04-unicamp/) workshop that we run in Campinas. The first one was an remote workshop last year.
+
+We had 40 seats available but we sold less than 20. At this workshop we charged R$100.00 (that at the time was something around US$25.00) per seat. [A friend complained that the ticket was expensive](https://twitter.com/mariojmaaz/status/720009962085294082) but the workshops aren't free of costs and if the host isn't covering the costs we need to charge the attendees.
+
+Not having a full room helped a lot when we needed to find another one since the reserved room wasn't available due an strike promoted by the Student Union.
+
+## Workshop in São Paulo
+
+[This](https://rgaiacs.github.io/2016-05-27-ccsl/) was our forth workshop in São Paulo in the last three years making the place in Brazil with more workshops so far. All this four workshops had the support of [The FLOSS Competence Center](http://ccsl.usp.br/) and I hope they will continue to support us.
+
+The 18 seats for this workshop sold out in a few days and we had a full room on all the sessions. This probably happend because our history of workshop lots of positive feedbacks. On the previous events we had a waiting list with the double of our capacity but it didn't happen this time because I did a very poor advertisement of this workshop that happened over a holiday.
```

Comment:
history of workshops... -> history of workshops has positive feedback

reword: "At previous events we had a waiting list filled to double the capacity, this time we did not because of less advertisement and the workshop was scheduled during a holiday."